core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -63,116 +63,118 @@ public Put(Put put) {
    *
    * @param value a {@code Value} to put
    * @return this object
+   * @deprecated As of release 3.6.0. Will be removed in release 5.0.0
    */
+  @Deprecated
   public Put withValue(Value<?> value) {
```

Comment:
I deprecated this `withValue(Value<?>)` method and the `withValues(List<Value<?>>)`.